.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# See Dependabot documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/dev/ci"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "jmagman"
+      - "keyonghan"
+      - "yusufm"
+    labels:
+      - "team"
+      - "team: infra"
+      - "waiting for tree to go green"
```

Comment:
Will add [dependabot](https://github.com/flutter/cocoon/blob/70ba45e6998175aa79c2ad912fb9e6e8c1ba3299/app_dart/lib/src/datastore/cocoon_config.dart#L246-L247) to the autorollers list so we can mostly ignore it if the tests pass.